From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 11 Oct 2021 13:55:35 +0200
Subject: broadway: Quiet initialisation failures

Broadway is the only GTK+ backend that throws an error on stderr for a
"display server" connection failure.

This causes problems when gtk_init_check() is used and unexpected error
output is generated such as with hotdoc, which fails when generating a
GTK plugin's documentation instead of overlooking the issue.

"Unable to init server: Could not connect: Connection refused"

Origin: upstream, 3.24.31, commit:e7679635b0cb94475ceab1b22f719721b8b0f6ac
---
 gdk/broadway/gdkdisplay-broadway.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdk/broadway/gdkdisplay-broadway.c b/gdk/broadway/gdkdisplay-broadway.c
index 54a1fa8..e71d6cb 100644
--- a/gdk/broadway/gdkdisplay-broadway.c
+++ b/gdk/broadway/gdkdisplay-broadway.c
@@ -109,7 +109,7 @@ _gdk_broadway_display_open (const gchar *display_name)
   broadway_display->server = _gdk_broadway_server_new (display_name, &error);
   if (broadway_display->server == NULL)
     {
-      g_printerr ("Unable to init server: %s\n", error->message);
+      GDK_NOTE (MISC, g_message ("Unable to init Broadway server: %s\n", error->message));
       g_error_free (error);
       return NULL;
     }
